os/arch/arm/src/common/up_initialize.c

```diff
@@ -285,7 +285,7 @@ void up_initialize(void)
 
 	/* Initialize USB -- device and/or host */
 
-	up_usbinitialize();
+	//up_usbinitialize();
```

Comment:
This is not only for stm, you should not comment this for other board.
Could you let me know the reason of commenting out and could you cover it with stm config if it should be disabled?